instat/ucrAxes.vb

```diff
@@ -197,25 +188,44 @@ Public Class ucrAxes
         AddRemoveXYScales()
     End Sub
 
-    Private Sub tickMarkersDisplay()
-        If rdoTickMarkersCustom.Checked AndAlso ucrTickMarkers.GetText = "Specific Values" Then
-            ucrSpecificValues.Visible = True
-            ucrInputFrom.Visible = False
-            lblFrom.Visible = False
-            ucrInputTo.Visible = False
-            lblTo.Visible = False
-            ucrInputInStepsOf.Visible = False
-            lblInStepsOf.Visible = False
-        Else
-            ucrSpecificValues.Visible = False
-            ucrInputFrom.Visible = True
-            lblFrom.Visible = True
-            ucrInputTo.Visible = True
-            lblTo.Visible = True
-            ucrInputInStepsOf.Visible = True
-            lblInStepsOf.Visible = True
-        End If
+    'Private Sub tickMarkersDisplay()
+    '    If rdoMajorBreaksNone.Checked AndAlso ucrTickMarkers.GetText = "Specific Values" Then
+    '        ucrInputMajorBreaksCustom.Visible = True
+    '        ucrInputMajorBreaksFrom.Visible = False
+    '        lblMajorBreaksFrom.Visible = False
+    '        ucrInputMajorBreaksTo.Visible = False
+    '        lblMajorBreaksTo.Visible = False
+    '        ucrInputMajorBreaksInStepsOf.Visible = False
+    '        lblMajorBreaksInStepsOf.Visible = False
+    '    Else
+    '        ucrInputMajorBreaksCustom.Visible = False
+    '        ucrInputMajorBreaksFrom.Visible = True
+    '        lblMajorBreaksFrom.Visible = True
+    '        ucrInputMajorBreaksTo.Visible = True
+    '        lblMajorBreaksTo.Visible = True
+    '        ucrInputMajorBreaksInStepsOf.Visible = True
+    '        lblMajorBreaksInStepsOf.Visible = True
+    '    End If
+    'End Sub
+
+    Private Sub ucrTickMarkers_ControlValueChanged(ucrChangedControl As ucrCore) Handles ucrPnlMajorBreaks.ControlValueChanged
+
     End Sub
 
+    Private Sub ucrInputAxisType_ControlValueChanged(ucrChangedControl As ucrCore) Handles ucrInputAxisType.ControlValueChanged
+        SetAxisTypeControls()
+    End Sub
+
+    Private Sub SetAxisTypeControls()
+        strAxisType = ucrInputAxisType.GetText()
+        'hide all axis type lanels
+        If strAxisType = "Continuous" Then
+            'show continous panels
+        ElseIf strAxisType = "Discrete" Then
+            'show discrete panels
+        ElseIf strAxisType = "Date" Then
+            'show date panels
+        End If
+    End Sub

```

Comment:
This sub needs to be filled in.
Currently the control only has controls for continuous scales. These need to be put in panels so that hiding/showing is easy here.
Then this method needs to be filled in.
Controls for discrete etc. can be added later, but it should hide the continuous controls when selected.